src/lib.rs

```diff
@@ -827,4 +854,14 @@ pub mod __rt {
         static FOO: AtomicUsize = ATOMIC_USIZE_INIT;
         FOO.store(0, Ordering::SeqCst);
     }
+
+    pub trait IsResult {
```

Comment:
Where is this being used? What is this for?

Reply:
Oops, left over from previous implementation